src/FileFormats/FileFormats.jl

```diff
@@ -51,11 +51,10 @@ The `filename` argument is only needed if `format == FORMAT_AUTOMATIC`.
 
 `kwargs` are passed to the underlying model constructor.
 """
-function Model(
-    ;
+function Model(;
     format::FileFormat = FORMAT_AUTOMATIC,
-    filename::Union{Nothing, String} = nothing,
-    kwargs...
+    filename::Union{Nothing,String} = nothing,
+    kwargs...,
```

Comment:
It's a bit weird to add a comma here, is that expected or a bug in the formatter?

Reply:
It seems to be used when it is multiline, it makes sense

Reply:
This is expected.